deb/usr/local/bin/ssl.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
-source /etc/ces/functions.sh
```

Comment:
This must not be deleted or functions like `get_domain` or `get_fqdn` will not work.